openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -135,27 +136,121 @@ $link-hover: $edx-blue-l1 !default; // from our Pattern Library http://ux.edx.or
   }
 
   .student__answer__display__content {
-    border: 1px solid rgba($color-decorative-staff, 0.25);
-    @include padding(($baseline-v/2), ($baseline-h/2), ($baseline-v/2), ($baseline-h/2));
+    @include padding(($baseline-v/2), 0, ($baseline-v/2), 0);
     margin-bottom: ($baseline-v/2);
   }
 
-  .openassessment__student-info_list {
+  .staff-info__student__report {
     list-style-type: none;
+
+    h2.title {
+      margin-top: ($baseline-v/2);
+      border-top: 1px solid $heading-staff-color;
+    }
   }
 
   .value {
     width: $max-width/2;
   }
 
-  /**
-   * The follow styles are bound for the "shame" file. This is done to override
-   * LMS specific styles on HTML elements.
-   */
+  .ui-staff__action__section {
+    padding: $baseline-v ($baseline-h/2);
+    background-color: white;
+    margin-bottom: ($baseline-v/2);
+
+    .ui-staff__title {
+      text-transform: none;
+      font-size: 18px;
+      font-weight: 600;
+      line-height: 27px;
+    }
+
+    .label {
+      color: $edx-gray-d4;
+    }
 
-  // 'p' elements in LMS have a color set on them.
-  .student__answer__display__content p {
-    color: inherit;
+    .title {
+      color: $edx-gray-d4;
+    }
+
+    .title--sub {
+      font-size: 18px;
```

Comment:
@andy-armstrong you added this CSS, but I don't see that there are any elements with class title--sub within class ui-staff__action__section. I suspect that this CSS is unnecessary. Can you take a look?